jobs/urls.py

```diff
@@ -3,6 +3,7 @@
 from . import views
 
 urlpatterns = patterns('',
+    url(r'^$', views.main, name='main'),
```

Comment:
There currently is no /community url (in the main djangogirls/urls.py); if we're adding jobs to the community page, let's also change the root /jobs to /community (then we have e.g.  /djangogirls/community/jobs rather than /djangogirls/jobs/jobs )